spec/scenarios/orphaned_lock.py

```diff
@@ -48,9 +44,5 @@ def test(
             slices, test_name, demo, tacos_branch, branch=2
         ) as pr2,
     ):
-
-        try:
```

Comment:
dont we want this to fail in this way? to make sure in the future we are failing this test in the right way?

Reply:
Wdym? In this test case the original PR was closed so plan is not meant to fail?

I assumed that was just there temporarily since the implementation wasn't done yet.

Reply:
@maxwelllwang This looks right to me -- the bug that caused this expected failure is now fixed, so the `raise XFailed` is gone.

Reply:
agree